src/qt_gui/translations/pl_PL.ts

```diff
@@ -1,491 +1,493 @@
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="95"/>
+        <source>Emulator Language</source>
+        <translation>Język emulatora</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="114"/>
+        <source>Emulator</source>
+        <translation>Emulator</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="122"/>
+        <source>Enable Fullscreen</source>
+        <translation>Włącz pełny ekran</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="129"/>
+        <source>Show Splash</source>
+        <translation>Pokaż ekran powitania</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="136"/>
+        <source>Is PS4 Pro</source>
+        <translation>Emulacja PS4 Pro</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="155"/>
+        <source>Username</source>
+        <translation>Nazwa użytkownika</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="178"/>
+        <source>Logger</source>
+        <translation>Dziennik zdarzeń</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="199"/>
+        <source>Log Type</source>
+        <translation>Typ dziennika</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="235"/>
+        <source>Log Filter</source>
+        <translation>Filtrowanie dziennika</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="272"/>
+        <source>Graphics</source>
+        <translation>Grafika</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="282"/>
+        <source>Graphics Device</source>
+        <translation>Karta graficzna</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="326"/>
+        <source>Width</source>
+        <translation>Szerokość</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="357"/>
+        <source>Height</source>
+        <translation>Wysokość</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="405"/>
+        <source>Vblank Divider</source>
+        <translation>Dzielnik pionowego blankingu (Vblank)</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="453"/>
+        <source>Advanced</source>
+        <translation>Zaawansowane</translation>
+    </message>
+    <message>
+        <location filename="../settings_dialog.ui" line="462"/>
+        <source>Enable Shaders Dumping</source>
+        <translation>Włącz zgrywanie tekstur</translation>
```

Comment:
```suggestion
        <translation>Włącz zgrywanie shaderów</translation>
```
There is small not-insignificant difference between shaders and textures.

Reply:
`shaders` are `cienie`, don't suggest random ass polished english words, please.
Not sure what I wrote `textures` o.0